scrapers/Boobpedia.yml

```diff
@@ -22,48 +22,72 @@ xPathScrapers:
   performerScraper:
     performer:
-      Name: //h1
-      Twitter: //table//tr/td/b/a[text()='Twitter']/@href
-      Instagram: //table//tr/td/b/a[text()='Instagram']/@href
+      Name: //h1     
+      Gender:
+        fixed: "Female"
+      Twitter: //table//tr/td/b[text()='Twitter']/../following-sibling::td//@href
+      Instagram: //table//tr/td/b[text()='Instagram']/../following-sibling::td//@href
       Birthdate:
-        selector: //table//tr/td//b[text()='Born:']/../following-sibling::td/a
+        selector: //table//tr/td//b[text()='Born']/../following-sibling::td/a
         concat: " "
         postProcess:
+          - replace:
+              - regex: (.*\d\d\d\d).*
+                with: $1
           - parseDate: January 2 2006
-      Ethnicity: //table//tr/td/b[text()='Ethnicity:']/../following-sibling::td/a
-      Country: //table//tr/td/b[text()='Nationality:']/../following-sibling::td/a
-      EyeColor: //table//tr/td/b[text()='Eye color:']/../following-sibling::td
+      Ethnicity: //table//tr/td/b[text()='Ethnicity']/../following-sibling::td/a
+      Country: //table//tr/td/b[text()='Nationality']/../following-sibling::td/a
+      EyeColor: //table//tr/td/b[text()='Eye color']/../following-sibling::td
       Height:
-        selector: //table//tr/td/b[text()='Height:']/../following-sibling::td
+        selector: //table//tr/td/b[text()='Height']/../following-sibling::td
         postProcess:
           - replace:
               - regex: (?:.+\D)?(\d+\.\d+)\Dm.+
                 with: $1
               - regex: \.
                 with: ""
       Weight:
-        selector: //table//tr/td/b[text()='Weight:']/../following-sibling::td
+        selector: //table//tr/td/b[text()='Weight']/../following-sibling::td
         postProcess:
           - replace:
               - regex: (?:.+\D)?(\d+)\Dkg.+
                 with: $1
       Measurements:
-        selector: //table//tr/td/b[text()='Measurements:']/../following-sibling::td|//table//tr/td[contains(b,'cup')]/following-sibling::td
+        selector: //table//tr/td/b[text()='Measurements']/../following-sibling::td|//table//tr/td[contains(b,'cup')]/following-sibling::td
         concat: "|"
         postProcess:
           - replace:
-              - regex: (\d+)-(\d+)-(\d+)[^|]+\|(\S+).+ # get measurements + cup
-                with: $1$4-$2-$3
+              - regex: (\d+)-(\d+)-(\d+)[^|]*\|(\S+).+ # get measurements + cup
+                with: $4-$2-$3
               - regex: \|.+$ # fallback to clear non matching regexes
                 with: ""
-      FakeTits: //table//tr/td/b[text()='Boobs:']/../following-sibling::td/a
-      HairColor: //table//tr/td[contains(b,'Hair')]/following-sibling::td
+              - regex: \[\d*\] # Remove References
+                with: ""
+              - regex: ( in) # Remove Unit Inches
+                with: ""
+      FakeTits: //table//tr/td/b[text()='Boobs']/../following-sibling::td/a
```

Comment:
```yaml
      FakeTits: 
        selector: //table//tr/td/b[text()='Boobs']/../following-sibling::td/a
        postProcess:
          - map:
              "Enhanced": "Fake"
              "Natural": "Natural"
```
Re-maps the values to keep them in line with StashDB.

Reply:
Agreed and implemented